openstudiocore/src/openstudio_lib/OSDocument.cpp

```diff
@@ -443,6 +445,18 @@ namespace openstudio {
 
     m_model = model;
 
+    // convert absolute weather file paths to relative in the model, also copy the epw file to the temp dir
+    // remove the weather file object if something goes wrong
+    bool weatherFileOk = this->fixWeatherFileInTemp(true);
+    if (!weatherFileOk){
+      // weather file in model was reset because the file did not exist or it was not a valid EPW
+      modified = true;
+
+      // connect to slot that would show user error dialog
+      // DLM: Evan todo
+      //QTimer::singleShot(0, this, SLOT(weatherFileReset()));
```

Comment:
@evanweaver this is where user warning would go, something like:

"Invalid weather file, weather file has been reset"